onsite/phosphors_scoring.py

```
@@ -0,0 +1,540 @@
+                if not hit.metaValueExists("PhosphoRS_site_probs"):
+                    hit.setMetaValue("PhosphoRS_site_probs", "{}")
+                if not hit.metaValueExists("SpecEValue_score") and hit.metaValueExists("MS:1002052"):
+                    hit.setMetaValue("SpecEValue_score", float(hit.getMetaValue("MS:1002052")))
+        IdXMLFile().store(out_file, protein_ids, peptide_ids)
+        print(f"Successfully saved {len(peptide_ids)} identifications to {out_file}")
+    except Exception as e:
+        print(f"Error saving results: {str(e)}")
+        traceback.print_exc()
+        sys.exit(1)
+
+def find_spectrum_by_mz(exp, target_mz, rt=None, ppm_tolerance=10):
```

Ruff (0.14.0)

Unused function argument: `rt`

(ARG001)